src/docs/previews/dropdown-menu/main/css/index.svelte

```diff
@@ -0,0 +1,323 @@
+	import { createDropdownMenu, melt } from '$lib/index.js';
+	import { writable } from 'svelte/store';
+	import { AlignJustify, ChevronRight, Check } from 'lucide-svelte';
+	import { fly } from 'svelte/transition';
+
+	const settingsSync = writable(true);
+	const hideMeltUI = writable(false);
+
+	const {
+		elements: { trigger, menu, item, separator, arrow },
+		builders: { createSubmenu, createMenuRadioGroup, createCheckboxItem },
+		states: { open },
+	} = createDropdownMenu({
+		forceVisible: true,
+	});
+
+	const {
+		elements: { subMenu, subTrigger },
+		states: { subOpen },
+	} = createSubmenu();
+
+	const {
+		elements: { radioGroup, radioItem },
+		helpers: { isChecked },
+	} = createMenuRadioGroup({
+		defaultValue: 'Hunter Johnston',
+	});
+
+	const {
+		elements: { checkboxItem },
+	} = createCheckboxItem({
+		checked: settingsSync,
+	});
+
+	const {
+		elements: { checkboxItem: checkboxItemA },
+	} = createCheckboxItem({
+		checked: hideMeltUI,
+	});
+
+	const personsArr = [
+		'Hunter Johnston',
+		'Thomas G. Lopes',
+		'Adrian Gonz',
+		'Franck Poingt',
+	];
+</script>
+
+<button
+	type="button"
+	class="trigger"
+	use:melt={$trigger}
+	aria-label="Update dimensions"
+>
+	<AlignJustify class="square-4" />
+	<span class="sr-only">Open Popover</span>
+</button>
+
+{#if $open}
+	<div class="menu" use:melt={$menu} transition:fly={{ duration: 150, y: -10 }}>
+		<div class="item" use:melt={$item}>About Melt UI</div>
+		<div class="item" use:melt={$item}>Check for Updates...</div>
+		<div class="separator" use:melt={$separator} />
+		<div class="item" use:melt={$checkboxItem}>
+			<div class="check">
+				{#if $settingsSync}
+					<Check class="square-4" />
+				{/if}
+			</div>
+			Settings Sync is On
+		</div>
+		<div class="item" use:melt={$subTrigger}>
+			Profiles
+			<div class="rightSlot">
+				<ChevronRight class="square-4" />
+			</div>
+		</div>
+		{#if $subOpen}
+			<div
+				class="menu subMenu"
+				use:melt={$subMenu}
+				transition:fly={{ x: -50, duration: 150 }}
+			>
+				<div class="text">People</div>
+				<div use:melt={$radioGroup}>
+					{#each personsArr as person}
+						<div class="item" use:melt={$radioItem({ value: person })}>
+							<div class="check">
+								{#if $isChecked(person)}
+									<div class="dot" />
+								{/if}
+							</div>
+							{person}
+						</div>
+					{/each}
+				</div>
+			</div>
+		{/if}
+		<div use:melt={$separator} class="separator" />
+
+		<div class="item" use:melt={$checkboxItemA}>
+			<div class="check">
+				{#if $hideMeltUI}
+					<Check class="square-4" />
+				{/if}
+			</div>
+			Hide Melt UI
+			<div class="rightSlot">⌘H</div>
+		</div>
+		<div class="item" use:melt={$item} data-disabled>
+			Show All Components
+			<div class="rightSlot">⇧⌘N</div>
+		</div>
+		<div use:melt={$separator} class="separator" />
+		<div class="item" use:melt={$item}>
+			Quit Melt UI
+			<div class="rightSlot">⌘Q</div>
+		</div>
+		<div use:melt={$arrow} />
+	</div>
+{/if}
+
+<style>
```

Comment:
Please include `* { all: unset; }` to ensure everything works the same independent of the outside context (e.g. CSS Resets, other classes, etc)

Reply:
Ok I see, I didn't want to write classes like `sr-only` and `square-4` but no worries I'll modify the styles.